src/app/core/admin/admin-dashboard/admin-dashboard.component.html

```diff
@@ -0,0 +1,109 @@
+<div class="header flex-row flex-wrap justify-space-between">
+  <app-view-title
+    class="admin-ui-title"
+    i18n="page title"
+    [displayInPlace]="true"
+  >
+    Edit Dashboard Widgets
+  </app-view-title>
+
+  <div class="flex-row gap-small save-buttons">
+    <button mat-raised-button color="accent" (click)="save()" i18n>Save</button>
+    <button mat-stroked-button (click)="cancel()" i18n>Cancel</button>
+  </div>
+</div>
+
+<div class="hint-banner" i18n>
+  You can customize the widgets on your dashboard to highlight the information
+  that is most useful to you. You can personalize the dashboard to show the key
+  insights you need.
```

Comment:
I feel like the two sentences here say more or less the same? Suggesting a little adjustment:


```suggestion
  Drag and drop the widgets to reorder them. You can configure details of each widget by clicking on them.
  Add additional widgets or remove widgets to fully customize the dashboard to your needs.
  The dashboard layout is shared by all users, your changes will be visible to everyone.
```